src/views/ContinuationIn/ContinuationInBusinessBc.vue

```diff
@@ -0,0 +1,235 @@
+<template>
+  <div id="continuation-in-business-bc">
```

Comment:
Missing some parts in this file? `Name` and `Folio/Reference Number(Optional)`

Reply:
Great catch Hongjing!

For the name, we have a ticket for that: https://github.com/bcgov/entity/issues/19266. However, I'll add a placeholder for that now.

As for Folio Number, the component is in the review and confirm page. However, I'll double check with Yui if we need it in two places.

Reply:
Added placeholder. 

Folio number is now in step 2 instead of step 5 (as per my conversation with Janis).